Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -116,7 +139,7 @@ public final class LoginToutViewModel: LoginToutViewModelType, LoginToutViewMode
           .materialize()
       }
 
-    self.logIntoEnvironment = facebookLogin.values()
+    self.attemptAppleLogin = self.appleLoginButtonPressedProperty.signal.ignoreValues()
```

Comment:
is it ok to get rid of `self.logIntoEnvironment = facebookLogin.values()` here? Assuming maybe we moved this somewhere else?

Reply:
We just moved it down within the `init` function, but still need it to login using Facebook. On the second PR (to be opened today), it will be used also to handle Apple login. i.e: 
```
self.logIntoEnvironment = Signal.merge(facebookLogin, appleLogin)
```